src/main/scala/com/codacy/client/bitbucket/PullRequest.scala

```diff
@@ -58,8 +58,9 @@ object PullRequest {
       (__ \ "destination" \ "repository" \ "full_name").read[String] and
       (__ \ "destination" \ "branch" \ "name").read[String] and
       (__ \ "destination" \ "commit" \ "hash").read[String] and
+      // TODO: (__ \ "destination" \ "commit" \ "hash").read[Option[String]] and
```

Comment:
null

On Sun, 29 May 2016 at 22:50, Pedro Rijo <notifications@github.com> wrote:

> In src/main/scala/com/codacy/client/bitbucket/PullRequest.scala
> <https://github.com/codacy/bitbucket-scala-client/pull/14#discussion_r65012182>
> :
>
> > @@ -58,8 +58,9 @@ object PullRequest {
> >        (__ \ "destination" \ "repository" \ "full_name").read[String] and
> >        (__ \ "destination" \ "branch" \ "name").read[String] and
> >        (__ \ "destination" \ "commit" \ "hash").read[String] and
> > +      // TODO: (__ \ "destination" \ "commit" \ "hash").read[Option[String]] and
>
> dest commit hash may not exist? what do we get now then, an empty string?
>
> —
>
>
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/codacy/bitbucket-scala-client/pull/14/files/8dc76f7e5051e724f365dc68ca85a80d92f4a3e3#r65012182>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APRsy5P7KlM8yWFyJccBj3qeaGcGPAHzks5qGgopgaJpZM4IpT9U>
> .
>